### PR TITLE
Feature/cpo 53 investigate multiple environments 

### PR DESCRIPTION
**Pull request by Polleke007**

### CODE REVIEW

File: contentful.config.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ const url = process.env.NEXT_PUBLIC_BASE_URL;
 module.exports = {
   contentful: {
     space_id: process.env.CONTENTFUL_SPACE_ID || '',
-    cda_token: process.env.CONTENTFUL_ACCESS_TOKEN || '',
-    cpa_token: process.env.CONTENTFUL_PREVIEW_ACCESS_TOKEN || '',
+    cda_token: process.env.CONTENTFUL_ACCESS_TOKEN_LOCAL || '',
```
Comment (koenkicken) [inline contentful.config.js:6]: why should this become local?

File: src/lib/fetchConfig.ts [hunk 2]
```diff
@@ -22,10 +35,21 @@ export function customFetcher<TData, TVariables extends { preview?: boolean | nu
   options?: RequestInit['headers'],
 ) {
   return async (): Promise<TData> => {
-    const res = await fetch(fetchConfig.endpoint as string, {
+    const env = process.env.NODE_ENV;
+    let endpoint = fetchConfig.localEndpoint;
+    let apiToken = fetchConfig.localParams;
+    if(env == "development"){
```
Comment (koenkicken) [inline src/lib/fetchConfig.ts:41]: fix formatting

File: contentful.config.js (hunk 1 above)
Comment (koenkicken) [reply contentful.config.js:6]: @Polleke007 shouldnt this also be dynamic? on production env you want CONTENTFUL_ACCESS_TOKEN, on lower env you want CONTENTFUL_ACCESS_TOKEN_LOCAL, no?

Comment (Polleke007) [reply contentful.config.js:6]: This is not used, and removed

**Pull request merged**